Added the Kadane Algorithm

pranav230 opened the pull request:
I have added the Kadane Algorithm as Day15 work as a file named Day15_Kadane_Algorithm.cpp

hardikhere commented:
> I have added the Kadane Algorithm as Day15 work as a file named Day12_Kadane_Algorithm.cpp

Hey glad you contributed, Can you please create a seprate folder for a topic on which kadane is based, and add kadanes algo in that, Please let me know if you are doing that. :) thanks

pranav230 commented:
> > I have added the Kadane Algorithm as Day15 work as a file named Day12_Kadane_Algorithm.cpp
> 
> Hey glad you contributed, Can you please create a seprate folder for a topic on which kadane is based, and add kadanes algo in that, Please let me know if you are doing that. :) thanks

Okay what should i name the folder?

pranav230 commented:
> > I have added the Kadane Algorithm as Day15 work as a file named Day12_Kadane_Algorithm.cpp
> 
> Hey glad you contributed, Can you please create a seprate folder for a topic on which kadane is based, and add kadanes algo in that, Please let me know if you are doing that. :) thanks

Added the folder for the Kadane.
